things/api/things/http/responses.go

```diff
@@ -243,6 +243,20 @@ func (res createConnectionsRes) Empty() bool {
 	return true
 }
 
+type createDisconnectionRes struct{}
```

Comment:
Same here, `disconnectListReq`, `disconnectionListRes` or something like that. `Create` just does not fit here well. Also, don't forget to add interface implementation check to the begging of this file: 
```go 
_ mainflux.Response = (*disconnectListRes)(nil)
```